yt_spoti_converter_django_application/views.py

```diff
@@ -1,5 +1,71 @@
 from django.shortcuts import render
+import os
+from django.shortcuts import redirect
+import json
+import requests
 
 
 def index(request):
-    return render(request, "index.html")
+    try:
+        # print("FOR DEBUG: ", os.environ.get("SPOTI_CLIENT_ID"))
+        if (
+            os.environ.get("SPOTI_CLIENT_ID") == None
+            or os.environ.get("SPOTI_CLIENT_SECRET") == None
+            or os.environ.get("YT_API_KEY") == None
+            or os.environ.get("DOMAIN_OR_IP") == None
+        ):
+            raise Exception("Enviroment variables not set!")
+
+        return render(request, "index.html")
+
+    except Exception as e:
```

Comment:
catching generic exceptions is most of the time bad & yields unexpected behavior (i.e. throwing errors under the rug and not being able to debug stuff properly)

also, why would you have a generic try except clause over an entire view?

I think there is something in the code complete book about catching exceptions as local as possible